tmca-tester/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
-    "node-menu": "^1.3.2"
+    "inquirer": "^12.5.0",
+    "node-menu": "^0.0.6"
```

Comment:
Question (non-blocking): Is `node-menu` still needed?

Reply:
Nope - good catch! This has been removed.